services/static-webserver/client/source/class/osparc/auth/ui/LoginView.js

```diff
@@ -89,7 +89,12 @@ qx.Class.define("osparc.auth.ui.LoginView", {
       createAccountBtn.addListener("execute", () => {
         createAccountBtn.setEnabled(false);
         if (config["invitation_required"]) {
-          if (osparc.product.Utils.getProductName().includes("s4l")) {
+          if (
+            osparc.product.Utils.isProduct("s4l") ||
+            osparc.product.Utils.isProduct("s4lacad") ||
+            osparc.product.Utils.isProduct("s4ldesktop") ||
+            osparc.product.Utils.isProduct("s4ldesktopacad")
+          ) {
```

Comment:
If this case is used many places in the UI then it might make sense to add a special method, e.g. `osparc.product.Utils.is_s4l_product()`

Reply:
For now, we need to keep ``s4l-lite`` out of the condition

Reply:
OK, makes sense. Thanks for the clarification